### DISCUSSION TITLE
Webdriverio slack service -When test fails, the total failed displayed as equal to 0

### DISCUSSION THREAD

**Discussion by ellatom** [webdriverio/webdriverio#8255] [Technical Discussion]
Hi, 
I have configured the slack service in wdio.conf.ts .
When I run tests  and test is failing I get in slack channel that 
* total failed equal  to zero
* total passed equal to zero
* total tests equal to zero.
Why does it not count the failed test? Please Help.
![image](https://user-images.githubusercontent.com/68844149/164160483-df7e2cf8-d6fb-436a-b174-056e2d80d761.png)
the configuration:
```
 [slack, {
      webHookUrl: "https://hooks.slack.com/services/X/X/X", 
      notifyOnlyOnFailure: false, 
      messageTitle: "========================" 
  }]
```
I tried to debug  package of webdriverio-slack-service and it looks like after I remove the following line from index.js file: 
`                // let testError = results.error.matcherResult.message().replace(/[\u001b\u009b][-[+()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-ORZcf-nqry=><]/g, "");
`
and updating this line of code 
`const attach = failedAttachment(test, "bug - count failed", results);`
The failed is shown in slack:
![image](https://user-images.githubusercontent.com/68844149/164164021-52d66dcb-c9d5-48aa-a5f2-eabdc2b56f8c.png)

It looks like the regex to show the error description in the slack is failing.
After more debugging it looks like the error occur because of the message- message isn't a function but a key in object after changing from message() to message it prints the error as well.
 `               let testError = results.error.matcherResult.message.replace(/[\u001b\u009b][-[+()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-ORZcf-nqry=><]/g, "");`

Can you update the package so that error won't occur
Thank, Ella

**Comment by erwinheitzman** [comment]
If this is a request for a bug fix then discussions isn't the place for this. Also, if you already have a working fix working locally then a contribution is a small step and certainly appreciated.

**Reply by ellatom** [reply]
@erwinheitzman Hi, I haven't contributed to any repository until now. What is the process to contribute fix? Is there a guideline? and If  I contribute -will it be added to package, so next time the package will be downloaded with this fix?
